core/iwasm/include/gc_export.h

```diff
@@ -910,7 +910,18 @@ wasm_runtime_pop_local_object_ref(wasm_exec_env_t exec_env);
  * @param n number to pop
  */
 WASM_RUNTIME_API_EXTERN void
-wasm_runtime_pop_local_object_refs(wasm_exec_env_t exec_env, uint32_t n);
+wasm_runtime_pop_local_obj_refs(wasm_exec_env_t exec_env, uint32_t n);
+
+/**
+ * get current local object ref from stack
```

Comment:
Could you change to `Get` as it is upper case in other places?

Reply:
aha, OK,  done!